pubspec.yaml

```diff
@@ -4,13 +4,13 @@ description: A pluggable, mockable process invocation abstraction for Dart.
 homepage: https://github.com/google/process.dart
 
 environment:
-  sdk: '>=2.14.0-0 <3.0.0'
+  sdk: '>=2.14.0-0 <4.0.0'
 
 dependencies:
-  file: '^6.0.0'
+  file: '^7.0.0'
```

Comment:
You may can use `file: '>=6.0.0 <8.0.0'` as there weren't any changes which had to be made and it provides much broader compatibility.